Dockerfile

```diff
@@ -24,3 +27,5 @@ COPY test/entrypoint.sh /entrypoint.sh
 
 # Code file to execute when the docker container starts up (`entrypoint.sh`)
 ENTRYPOINT ["/entrypoint.sh"]
+WORKDIR /app
+CMD ["snyk", "code test"]
```

Comment:
I don't think we want the default command for the image to be specific to `snyk` because this image is supposed to be used for different use cases. I think we can do without these two lines.

Reply:
OK. Removed them, thanks